frontend/src/components/Status.vue

```diff
@@ -31,5 +39,9 @@ export default {
 .online {
   background-color: $green;
 }
+
+.hidden {
```

Comment:
This class should be placed in "common" CSS file.

Reply:
Totally agree. I removed the class name entirely and used the same strategy as in the other places (conditional rendering based on props)